Dependency issue in react project

Unable to integrate this into existing react project, getting below error
```
Module not found: Error: Can't resolve './copilot-service' in '/Users/jyotip/Codebase/dopcumentation-opensource-docusaurus/node_modules/.pnpm/@blocksuite+presets@0.13.0-canary-202403160534-05a3c47_yjs@13.6.14/node_modules/@blocksuite/presets/dist/fragments/copilot-panel'
Did you mean 'index.js'?
BREAKING CHANGE: The request './copilot-service' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
[WARNING] {"moduleIdentifier":"/Users/jyotip/Codebase/dopcumentation-opensource-docusaurus/node_modules/.pnpm/babel-loader@9.1.3_@babel+core@7.24.0_webpack@5.90.3/node_modules/babel-loader/lib/index.js??ruleSet[1].rules[5].use[0]!/Users/jyotip/Codebase/dopcumentation-opensource-docusaurus/src/editor/editor.ts","moduleName":"./src/editor/editor.ts","loc":"1:491-500","message":"export 'Workspace' (imported as 'Workspace') was not found in '@blocksuite/store' (possible exports: ASTWalker, AssetsManager, AwarenessStore, BaseAdapter, BaseBlockTransformer, BaseReactiveYData, BlockModel, BlockSchema, BlockSnapshotSchema, BlockSuiteDoc, Boxed, CollectionInfoSnapshotSchema, Doc, DocCollection, DocSnapshotSchema, Generator, Job, MemoryBlobManager, ReactiveYArray, ReactiveYMap, Schema, Slice, SliceSnapshotSchema, Slot, Space, Store, Text, Utils, collectionMigrations, createAutoIncrementIdGenerator, createAutoIncrementIdGeneratorByClientId, createIndexeddbStorage, createMemoryStorage, createSimpleServerStorage, createYProxy, defineBlockSchema, docMigrations, fromJSON, getAssetName, internalPrimitives, isPureObject, nanoid, native2Y, popProp, sha, stashProp, toJSON, uuidv4, y2Native)"}
```

--- a/packages/presets/src/fragments/copilot-panel/copilot-panel.ts
+++ b/packages/presets/src/fragments/copilot-panel/copilot-panel.ts
@@ -1,6 +1,6 @@
 import './chat/chat.js';
 import './edgeless/edgeless.js';
-import './copilot-service.js';
+import './copilot-service/index.js';
 
 import { EdgelessComponentToolbar } from '@blocksuite/blocks';
 import { ShadowlessElement, WithDisposable } from '@blocksuite/lit';

--- a/packages/presets/src/fragments/copilot-panel/copilot-panel.ts
+++ b/packages/presets/src/fragments/copilot-panel/copilot-panel.ts
@@ -1,6 +1,6 @@
 import './chat/chat.js';
 import './edgeless/edgeless.js';
-import './copilot-service';
+import './copilot-service.js';
 
 import { EdgelessComponentToolbar } from '@blocksuite/blocks';
 import { ShadowlessElement, WithDisposable } from '@blocksuite/lit';

